src/database/migrations/20240501004030-create-size.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+'use strict';
+
+const sequelize = require('sequelize');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('sizes', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: sequelize.UUIDV4,
+        unique: true,
+      },
+      size_name: {
+        type: Sequelize.STRING,
+        allowNull: true,
+      },
+      size_number: { type: Sequelize.INTEGER, allowNull: true },
+      price: {
+        type: Sequelize.FLOAT,
+        allowNull: false,
+      },
+      quantity: {
+        type: Sequelize.INTEGER,
+        defaultValue: 1,
+      },
+      productId: {
+        type: Sequelize.UUID,
+        references: {
+          model: 'products',
+          key: 'id',
+        },
+        onDelete: 'CASCADE',
+        onUpdate: 'CASCADE',
```

Comment:
what if the product name is updated only, and nothing else to update, and setting onUpdate: 'CASCADE', means, it delete sizes related to that product after updating......